drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-client/src/test/java/org/drools/workbench/screens/dtablexls/client/editor/DecisionTableXLSEditorPresenterTest.java

```diff
@@ -91,11 +102,13 @@ public void setUp() throws Exception {
         presenter = new DecisionTableXLSEditorPresenter( view,
                                                          decisionTableXLSResourceType,
                                                          decisionTableXLSXResourceType,
-                                                         new ServiceMock() ) {
+                                                         busyIndicatorView, new ServiceMock()
+        ) {
             {
                 kieView = mock( KieEditorWrapperView.class );
-                versionRecordManager = mock( VersionRecordManager.class );
                 overviewWidget = mock( OverviewWidgetPresenter.class );
+                versionRecordManager = versionRecordManagerMock;
```

Comment:
You have to admire the ease it is to test sub-classes of ```KieEditor``` ;-) static initialiser blocks to set protected-level inherited fields in tests.. always nice to see!!